### PR TITLE
Add Declarations, Namespaces and Uses

### PR DESCRIPTION
**Pull request by GaryJones**
Pretty much copied from PSR-12, but adapted for WordPress.

### CODE REVIEW

File: php-coding-standards.md [hunk 1]
```diff
@@ -6,7 +6,104 @@ Keep the following points in mind when writing PHP code for WordPress, whether f
...
+<html>
+<body>
+    <?php
+        // ... additional PHP code ...
+    ?>
+</body>
+</html>
+```
+
+Declare statements MUST contain no spaces and MUST look like `declare(strict_types=1);`.
```
Comment (JDGrimes) [inline php-coding-standards.md]: Why no spaces for `declare`? This feels contrary to WordPress standards for everything else to me. "When in doubt, space it out." I'd probably space it like `declare( strict_types = 1 );`.

Comment (JDGrimes) [inline php-coding-standards.md]: > How about around the =? Will that still work?

Yes: https://3v4l.org/5FS6K

Comment (GaryJones) [inline php-coding-standards.md]: That `declare` was updated with spaces.

**Comment by GaryJones** [comment]
cc @drewapicture

**Pull request closed without merging**